src/NuGet.Core/NuGet.Versioning/NuGetVersion.cs

```diff
@@ -195,5 +195,13 @@ public bool IsSemVer2
         {
             get { return (_releaseLabels != null && _releaseLabels.Length > 1) || HasMetadata; }
         }
+
+        /// <summary>
+        /// Returns the original, non-normalized version string.
+        /// </summary>
+        public string OriginalVersion
```

Comment:
Nit: could be `public string OriginalVersion => _originalString` right? I like that syntax 😄